### ISSUE TITLE
Deleting messages by receiptHandle throws AWS.SimpleQueueService.InvalidBatchEntryId

### ISSUE THREAD

**Issue by mlilley** [TechnologyAdvice/Squiss#13]
When deleting messages by receiptHandle, aws throws:

> AWS.SimpleQueueService.InvalidBatchEntryId: A batch entry id can only contain alphanumeric characters, hyphens and underscores. It can be at most 80 letters long.

Message receipt handles are typically ~400 characters long, and when passing a string (receiptHandle) to deleteMessage(), items in the delete-batch are constructed with { Id: &lt;handle&gt;, receiptHandle: &lt;handle&gt; } (index.js, line 176).  

Aws doesn't seem to like the Id parameter being so long.  Perhaps one could invent Id's to use when deleting by receiptHandle (uuids, or incrementing integers?)

**Comment by TomFrost** [comment]
Hey mlilley, I'm so sorry this sat for so long. I've been unable to reproduce this-- I know it was quite awhile in the past, but do you have a script handy that illustrates this behavior?

**Comment by mlilley** [comment]
Hey.  Unfortunately I've moved on to other things, but I have managed to whip up a repro for you:

```
'use strict';
const Squiss = require('squiss');

const q = new Squiss({
    queueName: 'my-test-queue',
    awsConfig: { accessKeyId: 'XXX', secretAccessKey: 'YYY', region: 'ZZZ' }
});

q.sendMessage('test-message').then(() => {
    q.on('message', (msg) => q.deleteMessage(msg.raw.ReceiptHandle));
    q.start();
});
```

The problem lies here: https://github.com/TechnologyAdvice/Squiss/blob/master/src/index.js#L175-L177. 

When passing deleteMessage() a msg receipt handle instead of a msg object, you use this as the delete batch item id.  AWS doesn't like this.  The script above fails with:

```
AWS.SimpleQueueService.InvalidBatchEntryId: A batch entry id can only contain alphanumeric characters, hyphens and underscores. It can be at most 80 letters long.
```

To illustrate, msg.raw.ReceiptHandle is typically a string like this:

```
AZEB7Bq81Dlp9COfQLyrdozcPrZxAtC2Ctr66FJcJECEcKfN/X8lykNEFE3fdFrnmKYI4VNwh9aujzLeQh1Y7mGcE3IB+ZuwRUp4/ulD0Nj61ZcOA8DlCoKbXbqeUBPcaWanYcY0JSGtcLGfBjgFmy1S2+SjT47ZOn42eMPncYzcMG7LrpUQ5DtMn/KNTwolrrzDdtjGedh3AFX4ucHirpDcNvKKjKj505EKXNFBWwmO31JXWeHKlEOmk5zwZsQw7t6fB4+yh0C0bIW5L+ACuvikyCs54EyCYhf/OSE1iqD6BLFBTmwxXAddnSUnPNXxH820owQooMhuiCFBgDSQGK9MKZACyqisxGDm0GI6TeHMZYMw1lTGrb5ICJPeCnkp4oXJ5SvsBy+78skzzpbTvPzq5Q==
```

**Comment by TomFrost** [comment]
Thanks! This has been corrected in 1.1.0, publishing soon.
